frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -44,3 +46,16 @@
         </div>
     </div>
 </fieldset>
+
+@countryGroup.map{ cg =>
+    @if(cg.id.toLowerCase == "us") {
+        <div class="us-legal">
+            NOTICE REGARDING AUTOMATIC RENEWAL: IF YOU DO NOT CANCEL YOUR MEMBERSHIP PRIOR TO THE END OF YOUR INITIAL
```

Comment:
That question is going around again to double-check, but current understanding is that it's a legal requirement to be upper case in the US :(